### ISSUE TITLE
compiling error (TWBR)

### ISSUE THREAD

**Issue by Jobin7157** [Montvydas/MonteQuadcopter#1]
Whenever I try to compile the code I get a message "TWBR" was not declared in the scope

**Comment by ArabeJS** [comment]
yes me to. plz... how install this code make tuto show how install plz

**Comment by TheIcarus1010** [comment]
Hey hi
Could someone email the entire code to me on my personal email - zeanxavier22@gmail.com
I've assembled all the components , worth all my pocket money for this project😢
Only the code compilation seems to be a tedios task
Please i beg you to email it to me with entire libraries in one please😥😭😭😭

**Comment by Jobin7157** [comment]
I was trying to compile it in an nodemcu (WiFi development board) and that
was the root cause as it requires an external Bluetooth hardware having i2c
communication so if you are connecting it with Arduino than try taking help
from Arduino forum rather than GitHub
For the code that i was using it was the same as link given below
https://github.com/Montvydas/MonteQuadcopter

**Comment by AppleDog11011** [comment]
Everyone use the library that has been used for this code... In short download the 6yr old library
